test/index.test.js

```diff
@@ -16,8 +16,9 @@ describe('Server tests', () => {
     jest.unmock('../server/services/connection-service')
   })
 
-  test('createServer returns server', async () => {
+  test('createServer returns server', async (done) => {

```

Comment:
Alternatively, this test (and most others you've touched in this PR) could be written as a regular function, rather than `async`, since it doesn't await anything. That would remove the need for `done()` to be called at the end.

As far as I know it's just a matter of preference without a particularly strong argument either way. All async is quite nice for consistency, so the way you've done it would get my vote for now.

Reply:
I did it for consistency but you are right there is nothing to await in many of these tests.